### DISCUSSION TITLE
Joining slack community: @tabbyml.com email address required?

### DISCUSSION THREAD

**Discussion by jimlloyd** [TabbyML/tabby#1112] [General]
Tabby looks cool and I want to be able to try it out and ask questions. Slack would be great, but it seems I can't join your Slack community without a tabbyml.com email address. Is that just a misconfiguration error that should be fixed?

**Comment by wsxiaoys** [comment]
Hi, our previous invitation had got expired - please use https://links.tabbyml.com/join-slack , thank you!
